code/game/gamemodes/game_mode.dm

```diff
@@ -243,12 +237,47 @@
 							//			recommended_enemies if the number of people with that role set to yes is less than recomended_enemies,
 							//			Less if there are not enough valid players in the game entirely to make recommended_enemies.
 
+// Just the above proc but for alive players TODO: Refactor to not be copied code
```

Comment:
todo

Reply:
I'll be removing the TODO as I don't see a direct and good way to make this not just copied code